static/src/javascripts/projects/common/modules/commercial/braze/getMessageFromUrlFragment.ts

```diff
@@ -0,0 +1,65 @@
+import { log } from '@guardian/libs';
+import type { BrazeMessageInterface } from './brazeMessageInterface';
+
+const FORCE_BRAZE_ALLOWLIST = [
+	'preview.gutools.co.uk',
+	'preview.code.dev-gutools.co.uk',
+	'localhost',
+	'm.thegulocal.com',
+];
```

Comment:
Using an `as const` assertion can help, here. It will also prevent anyone from mutating the array.